content/de/about/we-are-using.md

```diff
@@ -1,314 +0,0 @@
----
-name: We Are Using
-title: Wer benutzt Preact?
```

Comment:
This doesn't look correct. This PR deletes the `We are using` page for every language. Maybe this change got included in this PR by accident? It's also the reason why the CI is failing.